tree.js

```diff
@@ -25,13 +24,12 @@ function treeMaker(tree){
   }
 }
 
-document.addEventListener("keypress", enterKeyPress);
-
 function enterKeyPress(keypress) {
 	if (keypress.which === 13) {
-    console.log("ioeroj");
-		treeMaker();
+    console.log("testing to see if keypress is working.");
```

Comment:
take this console.log out (your wouldn't have a test console.log like this in "production" like it is on master.